### DISCUSSION TITLE
Accidentally deleted the 'strimzi-cluster-operator-token' secret

### DISCUSSION THREAD

**Discussion by justin-silveri** [strimzi/strimzi-kafka-operator#5328] [General]
So the 'strimzi-cluster-operator-token' secret was accidentally deleted from our cluster. It recreated it right away, but the secret name changed. Now the strimzi pod is failing to start since it's looking for the previous secret that no longer exists. Are there any options to fix this issue without having to redeploy strimzi / recreate the kafka cluster?

**Comment by scholzj** [comment]
I guess you just need to delete the cluster operator pod?

**Reply by justin-silveri** [reply]
I thought the pod had already cycled, and that's why it was failing to come up. Deleting the pod did indeed rebuild it correctly, and I'm able to push updates to the cluster again. Thanks.

**Reply by scholzj** [reply]
I guess maybe the container just restarted, but the Pod didn't recreate with the new secret. Anyway, glad you got it working again.
